amethyst_network/README.md

```diff
@@ -1,10 +1,41 @@
 # Amethyst Networking
 
-The networking crate for the `amethyst` game engine. The main engine can be found at https://amethyst.rs.
+[![Build Status][s2]][l2] [![Latest Version][s1]][l1] [![docs.rs][s4]][l4] [![Join us on Discord][s5]][l5] [![MIT/Apache][s3]][l3]
 
-## Status
+[s1]: https://img.shields.io/crates/v/amethyst_network.svg
+[l1]: https://crates.io/crates/amethyst_network
+[s2]: https://jenkins.amethyst-engine.org/buildStatus/icon?job=amethyst%2Fmaster
+[l2]: https://jenkins.amethyst-engine.org/job/laminar/job/master/badge/icon
+[s3]: https://img.shields.io/badge/license-MIT%2FApache-blue.svg
+[l3]: docs/LICENSE-MIT
+[s4]: https://docs.rs/amethyst_network/badge.svg
+[l4]: https://docs.rs/amethyst_network/
+[s5]: https://img.shields.io/discord/425678876929163284.svg?logo=discord
+[l5]: https://discord.gg/GnP5Whs
 
-Right now this crate is very simple and it just supports sending messages through specs to other clients. It uses a combination of `EventChannel`s
-to send events between the network thread. In its current state it is not ready to be used in a game but there is work going into [Laminar](https://github.com/amethyst/laminar) that will soon be the underlying socket implemenation for this crate.
+The networking crate for the `amethyst` game engine. This crate provides the API and functionality which application developers will normally use to develop multiplayer games. The main engine can be found at https://amethyst.rs.
 
-For more information or help, please come find us on the discord server's #net channel.
+This project is at an early stage. We are currently designing and working on some robust, fast, distributed, networking system on top of specs. To make this work we are creating a small test game so that we can experiment with different solutions. Once in a while, we will move some stable functionality over from that game to amethyst network.
```

Comment:
"This project is still at an early stage. We are currently designing and implementing a fast/robust networking system on top of specs. To exercise our implementation, we are creating a small test game which we will make public when we feel it's in a good place. Eventually, as we gain more confidence in our solution, we will move stable functionality over from that game to amethyst network."

Reply:
Or something to that effect.

Reply:
Good one!